### PR TITLE
DBZ-3885 change UUID handling

### PR DESCRIPTION
**Pull request by fuxiao224**
change UUID representation from base64 encoding to standard string.

### CODE REVIEW

**Comment by fuxiao224** [comment]
Hi @bingqinzhou ! Would you please help me take a look at this PR? Please note that I noticed that there's no hyphen ('-') after convert UUID to string. For example, if input UUID: 000e0000-5719-12a3-0000-000028327d4a, it would output '000e0000571912a30000000028327d4a' in Kafka message. I investigated about this and seems like this is expected if we store UUID as byte array. Thanks!

File: src/main/java/io/debezium/connector/cassandra/transforms/type/deserializer/UUIDTypeDeserializer.java [hunk 1]
```diff
@@ -31,6 +30,10 @@ public SchemaBuilder getSchemaBuilder(AbstractType<?> abstractType) {
     @Override
     public Object formatDeserializedValue(AbstractType<?> abstractType, Object value) {
         byte[] bytes = UuidUtil.asBytes((java.util.UUID) value);
-        return Values.convertToString(getSchemaBuilder(abstractType).build(), bytes);
+        StringBuilder sb = new StringBuilder();
+        for (byte b : bytes) {
+            sb.append(String.format("%02x", b));
+        }
+        return sb.toString();
```
[the remark this answers is not in the document]
Comment (fuxiao224) [reply src/main/java/io/debezium/connector/cassandra/transforms/type/deserializer/UUIDTypeDeserializer.java:37]: Thanks! Will investigate more on this!

[the remark this answers is not in the document]
Comment (fuxiao224) [reply src/main/java/io/debezium/connector/cassandra/transforms/type/deserializer/UUIDTypeDeserializer.java]: Hi @bingqinzhou! Thanks for your suggestion, it makes sense now! I tried all 3 methods above. 1 & 2 doesn't seem to work very well. Method 3 works well, and the hyphen is included now. Would you please take a look again? Thanks!

**Comment by bingqinzhou** [comment]
LGTM, let's wait for @gunnarmorling's approval as well

**Comment by gunnarmorling** [comment]
Hey @fuxiao224, @bingqinzhou, no need for you to wait with these things on me; I trust you to do the right thing :) It LGTM, so will merge. Could you only send a PR for also updating the connector docs in the main repo accordingly. Thanks!

**Comment by fuxiao224** [comment]
> Hey @fuxiao224, @bingqinzhou, no need for you to wait with these things on me; I trust you to do the right thing :) It LGTM, so will merge. Could you only send a PR for also updating the connector docs in the main repo accordingly. Thanks!

Hi @gunnarmorling ! Thanks for reviewing and merging it! 
According to current doc:

Cassandra Data Type   Literal Type   Semantic Type
timeuuid                         string                  io.debezium.data.Uuid
uuid                                string                  io.debezium.data.Uuid

I think this is still true and nothing need to be changed?

**Comment by gunnarmorling** [comment]
Yes, I think that's right.

**Pull request merged**